src/main/resources/static/js/page/letsparty/detail.js

```diff
@@ -0,0 +1,186 @@
+let postNo = $("#postNo").val();
+
+// 최초 상세화면 이동시 댓글 2개 불러옴
+$(function() {
+	fetchAndRenderLatestTwoComments();
+});
+
+// 댓글 더보기 버튼 클릭시 전체 댓글 렌더링, 한 번 더 클릭하면 다시 최신 댓글 2개를 표시
+$(".more-button").click(function() {
+    let commentContainer = $("#comment-container");
+    if (commentContainer.hasClass("two-show")) { 
+		commentContainer.removeClass("two-show").addClass("all-show");
+	 	fetchAndRenderComments();
+		$(this).toggleClass("bi-chevron-down bi-chevron-up");
+	} else if(commentContainer.hasClass("all-show")) {
+		commentContainer.removeClass("all-show").addClass("two-show");
+		fetchAndRenderLatestTwoComments();
+		$(this).toggleClass("bi-chevron-down bi-chevron-up");
+	}
+});
+
+// 댓글쓰기 버튼을 누르면 댓글 등록 폼위치로 스크롤 이동
+$("#write-comment-btn").click(function(e) {
+	e.preventDefault();
+    $('html, body').animate({
+        scrollTop: $("#comment-form").offset().top
+    });
+});
+
+// 최근 댓글 2개만 불러오는 ajax 요청
+function fetchAndRenderLatestTwoComments() {
+    $.ajax({
+        url: "/letsparty/post/" + postNo + "/latest-two-comments",  // 최신 댓글 2개를 가져올 서버의 주소
```

Comment:
```suggestion
        url: `/letsparty/post/${postNo}"/latest-two-comments`,  // 최신 댓글 2개를 가져올 서버의 주소
```
ES6에 도입된 템플릿 리터럴 표기도 좋습니다.